apps/site/pages/en/about/previous-releases.mdx

```diff
@@ -14,12 +14,28 @@ Production applications should only use _Active LTS_ or _Maintenance LTS_ releas
 
 ![Releases](https://raw.githubusercontent.com/nodejs/Release/main/schedule.svg?sanitize=true)
 
-Full details regarding Node.js release schedule are available [on GitHub](https://github.com/nodejs/release#release-schedule).
+Full details regarding the Node.js release schedule are available [on GitHub](https://github.com/nodejs/release#release-schedule).
 
 ### Commercial Support
 
-Commercial support for versions past Maintenance phase is available through our OpenJS Ecosystem Sustainability Program partner [HeroDevs](https://herodevs.com/).
+Commercial support for versions past the Maintenance phase is available through our OpenJS Ecosystem Sustainability Program partner [HeroDevs](https://herodevs.com/).
 
-## Looking for latest release of a version branch?
+## Looking for the latest release of a version branch?
 
 <DownloadReleasesTable />
+
+## Official versus Community
+
+The Node.js website offers numerous installation methods that allow Node.js to be installed in a non-interactive manner,
+for example, via CLIs, OS package managers (such as `apt`), or Node.js version managers (such as `nvm`).
+
+The Node.js project, in an attempt to popularize and advertise community efforts, has introduced a new Downloads page that lists both Official and Community installation methods, providing more versatility and options for users.
+With this change, we introduced the concept of "Official" and "Community" installation methods. In order to be considered "Official", the installation method must meet the following requirements:
+
+| Requirements                                                                                          |
+| ----------------------------------------------------------------------------------------------------- |
+| New Node.js releases must be available simultaneously upon the official release                       |
```

Comment:
This is problematic for Docker images. They are considered official but don't comply with this requirement

Reply:
What is the grace period for Docker images? I believe simultaneously might be a harsh requirement. What if we say within one hour of release?

Reply:
I think that allowing a gap is going to open the floodgates too wide, and create a monitoring burden for the node project.

I think it's fine to have docker as an exception since it's the node project *itself* that provides it.

Reply:
I'm fine either way; I believe this is a decision for the Build WG and the Release Team.

Reply:
I'd be +1 to Docker being an exception to this -- I'd view this more as requirements for third-party but "official" methods, rather than first-party distribution methods (though if this can poke the nest to make the Docker process even faster/more automated ;))

Reply:
I have a long-standing todo to document the docker release process on the blog or somewhere...so we have a URL we can point to when people ask.  see threads like https://openjs-foundation.slack.com/archives/C019Y2T6STH/p1731922406905349 for example